### ISSUE TITLE
Name of room owner in room chat

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#325]
```
What steps will reproduce the problem?
1. Join the room.
2. In chat appears "%MyName has joined the room".
and "%OwnerName has joined the room".

What is the expected output? What do you see instead?
It chat should be written owner name like: "%OwnerName is owner." or something 
similar.

What version of the product are you using? On what operating system?
0.9.17
```

-----
Original issue reported on code.google.com by `blackmet...@o2.pl` on 4 Dec 2011 at 12:54

**Comment by GoogleCodeExporter** [comment]
```
Simple change though.

```

-----
Original comment by `kyberneticist@gmail.com` on 28 Jun 2012 at 6:32
* Added labels: **Type-Enhancement**, **Priority-Low**
* Removed labels: **Type-Defect**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in revision 4cb423f42105
```

-----
Original comment by `unC0Rr` on 4 Sep 2012 at 11:27
* Changed state: **Fixed**
